tests/unit/runner.js

```diff
@@ -8,7 +8,6 @@ import { initServer } from './server';
 
 nconf
   .argv()
-  .env()
```

Comment:
Unit tests don't need to care about the env, in fact the env variables being set in circle CI broke unit tests.